modules/troubleshooting-disabling-autoreboot-mco-cli.adoc

```diff
@@ -9,7 +9,7 @@ To avoid unwanted disruptions from changes made by the Machine Config Operator (
 
 [NOTE]
 ====
-Pausing an MCP stops all updates to your {op-system} nodes, including updates to the operating system, security, certificate, as well as any other updates related to the machine config. Pausing should be done for short periods of time only.
+Pausing a machine config pool prevents the Machine Config Operator from applying any configuration changes on the associated nodes. Pausing an MCP also prevents any automatically-rotated certificates from being pushed to the associated nodes, including the automatic CA rotation of the `kube-apiserver-to-kubelet-signer` CA certificate. If the MCP is paused when the `kube-apiserver-to-kubelet-signer` CA certificate expires and the MCO attempts to automatially renew the certificate, the new certificate is created but not applied across the nodes in the respective machine config pool. This causes failure in multiple `oc` commands, including but not limited to `oc debug`, `oc logs`, `oc exec`, and `oc attach`. Pausing an MCP should be done with careful consideration about the `kube-apiserver-to-kubelet-signer` CA certificate expiration and for short periods of time only.
```

Comment:
Question: does this apply only for master pool? Or is it both master and worker pools?

i.e. does the worker certs need to be updated for `oc` commands to function?

Reply:
@sttts @ravitri Can you help with @yuqi-zhang question ^^?

Reply:
> i.e. does the worker certs need to be updated for `oc` commands to function?

yes, I think it should due to mentioned commands having issues.